dencam/buttons.py

```diff
@@ -70,7 +70,7 @@ def run(self):
         GPIO.cleanup()
 
     def _set_screen_brightness(self):
-        if self.state.value > 0:
+        if self.state.value >= 0:
```

Comment:
This isn't what we want.  We want the zero state to turn off the screen backlight. This came up in the README too: the device is meant to start with a black screen and with the backlight off.  There are actually 5 states: that blank state with backlight off, Network page, Recording Page, Solar Page, and Video Preview page (which as you noted in the README is overlayed on a Blank page).